portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
```

Comment:
Please verify formatting.

Reply:
*This is for every file that you told me to run the formatter on

I'm not sure if I'm running the command wrong, but if I make the changes that it is telling me to my files no longer compile when I try to run my website. It's telling me to remove the package statement at the top and various import statements that are used throughout the file, so I am not sure what I am should do to resolve that.

Reply:
Is there an option I should specify when running the command so it won't tell me to delete the import statements?

Reply:
Is it asking you to completely delete the imports or is it asking you to reorder the import statements?  It would look like deleting them from the existing lines and adding them elsewhere.

Reply:
Actually I think it was the size of my terminal. When I was trying to enlarge it so I could take a screenshot, I noticed that more import statements started appearing in the formatted version of my file. I will make the full suggested changes and let you know if I encounter any errors.

Reply:
I was able to make the Java formatting changes by taking a picture of the enlarged terminal on my phone. I will look into the <options> portion of the command to see if there is a way of preserving the formatting even if i scroll up, but for now my files were still able to be compiled with the changes.